lib/Data/Validate/Sanctions.pm

```diff
@@ -106,11 +108,13 @@ sub get_sanctioned_info {    ## no critic (RequireArgUnpacking)
     my $matched_name;
     my $matched_file;
     my $dob_missing;
+    my $dob_other;
 
     for my $file (sort keys %$data) {
 
         my @names = keys %{$data->{$file}->{names_list}};
-
+        
+        $dob_other = [];
```

Comment:
Should we name it like `dob_incorrect` or something similar? to give clear information that this date has invalid format.

Reply:
I agree. What about `dob_text` (implying that it contains raw-text info)?

Reply:
It was a redundant variable. I just removed it.